sat/cli/bootsys/platform.py

```diff
@@ -484,7 +484,19 @@ def do_recreate_cronjobs(_):
     """Recreate cronjobs that are not being scheduled on time."""
     try:
         batch_api = load_kube_api(api_cls=BatchV1Api)
-        recreate_namespaced_stuck_cronjobs(batch_api, 'services')
+        # Adding a retry
+        max_retries = 3
+        retries = 0
+        while retries < max_retries:
+            try:
+                recreate_namespaced_stuck_cronjobs(batch_api, 'services')
+                break  # Exit the loop if successful
+            except Exception as e:
```

Comment:
This is a very broad `except` clause. If possible you should narrow it to the actual exception seen in the bug (or perhaps a parent class of that exception).